tests/suites/test_suite_psa_crypto.function

```diff
@@ -120,6 +203,64 @@ static int construct_fake_rsa_key( unsigned char *buffer,
     return( len );
 }
 
+int exercise_mac_setup( psa_key_type_t key_type,
+                        const unsigned char *key_bytes,
+                        size_t key_length,
+                        psa_algorithm_t alg,
+                        psa_mac_operation_t *operation,
+                        psa_status_t *status )
+{
+    psa_key_handle_t handle = 0;
+    psa_key_policy_t policy = PSA_KEY_POLICY_INIT;
+
+    PSA_ASSERT( psa_allocate_key( &handle ) );
+    psa_key_policy_set_usage( &policy, PSA_KEY_USAGE_SIGN, alg );
```

Comment:
We have coverage with different policies in `mac_policy`.

Since this function only tests `psa_mac_sign_setup` and not `psa_mac_verify_setup`, it only needs a `USAGE_SIGN` policy, just like the corresponding cipher test calls `cipher_encrypt_setup` with a `USAGE_ENCRYPT` policy. I think having both `USAGE_SIGN` and `USAGE_VERIFY` here was a leftover from the old days when MAC functions didn't check policies correctly.